examples/vacanza.py

```
@@ -0,0 +1,153 @@
+                categories=category,
+            )
+
+            if not country_holidays:
+                print(f"  No holidays found for category: {category}")
+                continue
+
+            file_name = f"{country_code}_{category}.ics"
+            ICalExporter(country_holidays).save_ics(file_name)
+            print(f"  ✅ Saved: {file_name} ({len(country_holidays)} holidays)")
+
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)